doc/manual/source/command-ref/nix-channel.md

```diff
@@ -53,6 +53,11 @@ This command has the following operations:
   Download the Nix expressions of subscribed channels and create a new generation.
   Update all channels if none is specified, and only those included in *names* otherwise.
 
+  > **Note**
+  >
+  > `--update` uses [fetchTarball](@docroot@/language/builtins.md#builtins-fetchTarball) under the hood, so it will cache channels.
+  > Use `--tarball-ttl` or the nix configuration option `tarball-ttl` to change this behavior.
```

Comment:
```suggestion
  > Use `--tarball-ttl` or the [`tarball-ttl` configuration option](@docroot@/command-ref/conf-file.md#conf-tarball-ttl) to change the validity period of cached downloads.
```

Linking to the relevant reference docs would be crucial here, otherwise there's no way to know how to use that option (except searching manually, which we should avoid at all costs).